### PR TITLE
Add getNetwork method to Provider

### PR DESCRIPTION
**Pull request by Ben-Rey**

### CODE REVIEW

File: src/provider/Provider.ts [hunk 1]
```diff
@@ -146,6 +146,31 @@ export class Provider implements IProvider {
     });
   }
 
+  // Get network
+  /**
+   * This method sends a message to the content script to get the used network name.
```
Comment (gregLibert) [inline src/provider/Provider.ts:151]: Same as above

File: unit-tests/code-snippets/get-nework.spec.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { providers } from '@massalabs/wallet-provider';
+
+const networks = ['mainnet', 'testnet', 'buildnet', 'labnet'];
```
Comment (gregLibert) [inline unit-tests/code-snippets/get-nework.spec.ts:3]: This variable doesn't make sense if the test is removed. If you want to use it as code snippet, you should remove it and directly use the array in the expect.

**Review by gregLibert — CHANGES_REQUESTED** [review]

File: unit-tests/code-snippets/get-nework.spec.ts (hunk 2 above)
Comment (Ben-Rey) [reply unit-tests/code-snippets/get-nework.spec.ts]: Thanks, I updated the example and added the code snippet like the generate-new-account test

**Review by gregLibert — CHANGES_REQUESTED** [review]

File: code-snippets/get-network.ts
[the remark this answers is not in the document]
Comment (Ben-Rey) [reply code-snippets/get-network.ts:1]: Should be ok thanks

**Review by gregLibert — APPROVED** [review]

**Pull request merged**